.github/workflows/ci.yml

```diff
@@ -12,27 +12,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8.2.0
       - uses: actions/setup-node@v3
         with:
           node-version: 16
-
-      - name: Get yarn cache
-        id: yarn-cache
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-
-      - name: Cache dependencies
-        uses: actions/cache@v1
-        with:
-          path: ${{ steps.yarn-cache.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
+          cache: 'pnpm'
+          cache-dependency-path: '**/pnpm-lock.yaml'
 
       - name: Install dependencies
-        run: yarn install
+        run: pnpm install --frozen-lockfile
 
       - name: Run linter
-        run: yarn run lint
+        run: pnpm run lint
 
       - name: Test
-        run: yarn run test
+        run: pnpm run test
```

Comment:
I would like to have as closely same as possible with others: https://github.com/jotaijs/jotai-trpc/blob/main/.github/workflows/ci.yml is the latest one.

Reply:
Sure! Pushed the change in [this commit](https://github.com/jotaijs/jotai-devtools/pull/65/commits/7f607db391429f28877134bacc3a8e379292d9a6). 

Out of curiosity, why do we do `npm run test` instead of `pnpm run test`? Is it because `npm` would always be available?

Reply:
Yeah, kind of. We can use `pnpm` as it's surely installed. I'm curious if there's any difference in CI.